lib/child_process.js

```diff
@@ -144,12 +140,12 @@ function normalizeExecArgs(command, options, callback) {
 }
 
 
-exports.exec = function exec(command, options, callback) {
+function exec(command, options, callback) {
   const opts = normalizeExecArgs(command, options, callback);
-  return exports.execFile(opts.file,
-                          opts.options,
-                          opts.callback);
-};
+  return execFile(opts.file,
```

Comment:
Note that these changes likely make this a semver-major change because it impacts the ability to monkeypatch the child_process module.

Reply:
I think it's the only line that has this impact. I'd prefer to replace with `module.exports.execFile` to avoid future conflicts in backports.

Reply:
resolved